src/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.cs

```diff
@@ -1091,5 +1092,7 @@ private void SetFlagToAddListSeparatorBeforeNextItem()
         {
             _currentDepth |= 1 << 31;
         }
+
+        private string DebuggerDisplay => $"BytesCommitted = {BytesCommitted} BytesPending = {BytesPending} CurrentDepth = {CurrentDepth}";
```

Comment:
How about last token type just written?

Reply:
I didn't put any non-public data on. Not sure if we've ever done that?

Reply:
Hmm, I am wondering if `TokenType` should be a public property, especially when we have public APIs/Converters that take a `Utf8JsonWriter`. It might be useful for implementer to know what has just been written last to know what operations are valid.

For example, if the writer is at `StartObject`, you know that `WritePropertyAndValue` would succeed, but at `StartArray`, it would not and the user can add checks for that.